### ISSUE TITLE
The accuracy is very low when executing deepcnn MNIST test.

### ISSUE THREAD

**Issue by jx991220** [zama-ai/concrete-ml#1041]
I tested DeepCNN-20/50/100 by executing benchmarks/deep_learning.py
but no matter I chose 20, 50 or 100, the plaintext accuracy was very low (only 8.2%), and even the ciphertext accuracy was only 0. 
I hope someone can tell me if I'm doing it wrong or if there's something else wrong?

As follows:
`python3 deep_learning.py --models Deep100CNN`

+---------------------------------------------------------+--------------------+
| Torch fp32 Accuracy                                     |           0.082222 |
+---------------------------------------------------------+--------------------+
| Torch fp32 MCC                                          |           0.000000 |
+---------------------------------------------------------+--------------------+
| Circuit bit width                                       |           5.000000 |
+---------------------------------------------------------+--------------------+
| Quantized Clear Accuracy                                |           0.082222 |
+---------------------------------------------------------+--------------------+
| Quantized Clear MCC                                     |           0.000000 |
+---------------------------------------------------------+--------------------+
| Quantized Clear (FHE set) Accuracy                      |           0.000000 |
+---------------------------------------------------------+--------------------+
| Quantized Clear (FHE set) MCC                           |           0.000000 |
+---------------------------------------------------------+--------------------+
| FHE Key Generation Time                                 |           2.582845 |
+---------------------------------------------------------+--------------------+
| Execution Time per sample for FHE                       |           2.400608 |
+---------------------------------------------------------+--------------------+
| FHE Accuracy                                            |           0.000000 |
+---------------------------------------------------------+--------------------+
| FHE MCC                                                 |           0.000000 |
+---------------------------------------------------------+--------------------+
| MAE score between clear and FHE quantized probabilities |           0.000000 |
+---------------------------------------------------------+--------------------+

**Comment by jfrery** [comment]
Hi @jx991220,

Thanks for reporting this. Indeed something is wrong here. The pre-trained model outputs ~10% accuracy which is basically what a random model would get. 0% accuracy is somewhat better than random so it's also wrong.

We will look into it.

**Comment by jx991220** [comment]
Thanks for the quick reply. Is it because of the code, not my usage? If so, please let me know after the code problem is fixed. Thanks!

**Comment by jfrery** [comment]
It is fixed in #1119. Somehow the .pt model checkpoint were corrupted at some point. Should be good once the PR is merged
